test/unit/resources/kernel_module_test.rb

```diff
@@ -11,18 +11,48 @@
     _(resource.loaded?).must_equal true
   end
 
-  it 'verify kernel_module parsing' do
+  it 'verify kernel_module parsing loaded' do
     resource = load_resource('kernel_module', 'bridges')
     _(resource.loaded?).must_equal false
   end
 
   it 'verify kernel_module parsing' do
+    resource = load_resource('kernel_module', 'bridge')
+    _(resource.enabled?).must_equal true
+  end
+
+  it 'verify kernel_module parsing' do
     resource = load_resource('kernel_module', 'dhcp')
     _(resource.loaded?).must_equal false
   end
 
+  it 'verify kernel_module parsing disabled' do
+    resource = load_resource('kernel_module', 'bridge')
+    _(resource.disabled?).must_equal false
+  end
+
+  it 'verify kernel_module parsing disabled' do
```

Comment:
This looks like a duplicate test to the one in lines 29-32, just with a different module name? We can remove this.